.github/actions/spelling/excludes.txt

```diff
@@ -79,6 +103,24 @@
 ^src/AppInstallerCLITests/TestData/InputPublishers.txt$
 ^src/AppInstallerCLITests/TestData/NormalizationInitialIds.txt$
 ^src/AppInstallerCommonCore/external/do.h$
+^\Qsrc/AppInstallerCLIE2ETests/TestData/Configuration/Empty.yml\E$
+^\Qsrc/cpprestsdk/cpprestsdk/Release/samples/BlackJack/BlackJack_UIClient/Cards.PNG\E$
+^\Qsrc/PowerShell/ExternalModules/PackageManagement/1.4.8.1/DSCResources/MSFT_PackageManagement/MSFT_PackageManagement.psm1\E$
+^\Qsrc/PowerShell/ExternalModules/PackageManagement/1.4.8.1/DSCResources/PackageManagementDscUtilities.psm1\E$
+^\Qsrc/PowerShell/ExternalModules/PackageManagement/1.4.8.1/PackageManagement.format.ps1xml\E$
+^\Qsrc/PowerShell/ExternalModules/PackageManagement/1.4.8.1/PackageManagement.psd1\E$
+^\Qsrc/PowerShell/ExternalModules/PackageManagement/1.4.8.1/PackageManagement.psm1\E$
+^\Qsrc/PowerShell/ExternalModules/PackageManagement/1.4.8.1/PackageManagement.Resources.psd1\E$
+^\Qsrc/PowerShell/ExternalModules/PackageManagement/1.4.8.1/PackageProviderFunctions.psm1\E$
+^\Qsrc/PowerShell/ExternalModules/PowerShellGet/2.2.5/DscResources/MSFT_PSModule/en-US/MSFT_PSModule.strings.psd1\E$
+^\Qsrc/PowerShell/ExternalModules/PowerShellGet/2.2.5/DscResources/MSFT_PSModule/MSFT_PSModule.psm1\E$
+^\Qsrc/PowerShell/ExternalModules/PowerShellGet/2.2.5/DscResources/MSFT_PSModule/MSFT_PSModule.schema.mof\E$
+^\Qsrc/PowerShell/ExternalModules/PowerShellGet/2.2.5/DscResources/MSFT_PSRepository/en-US/MSFT_PSRepository.strings.psd1\E$
+^\Qsrc/PowerShell/ExternalModules/PowerShellGet/2.2.5/DscResources/MSFT_PSRepository/MSFT_PSRepository.psm1\E$
+^\Qsrc/PowerShell/ExternalModules/PowerShellGet/2.2.5/DscResources/MSFT_PSRepository/MSFT_PSRepository.schema.mof\E$
+^\Qsrc/PowerShell/ExternalModules/PowerShellGet/2.2.5/PowerShellGet.psd1\E$
+^\Qsrc/PowerShell/ExternalModules/PowerShellGet/2.2.5/PSGet.Format.ps1xml\E$
+^\Qsrc/PowerShell/ExternalModules/PowerShellGet/2.2.5/PSModule.psm1\E$
```

Comment:
Could this be a single line up to ExternalModules to exclude the whole directory?

Reply:
Yes